source/_integrations/modbus.markdown

```diff
@@ -993,12 +993,24 @@ sensors:
       description: Unit to attach to value.
       required: false
       type: string
+    min_value:
+      description: The minimum allowed value of a sensor. If value < min_value --> min_value. Can be float or integer
+      required: false
+      type: float
```

Comment:
how do I say float | integer ??

Reply:
no idea, do a grep on the other integrations.

Reply:
I have confirmed from dev forum that I should set the type to float only.
Do you wan the comment 'Can be float or integer'

Reply:
It should be `float`, an integer is a valid float in Python.